docker/docker-entrypoint.sh

```diff
@@ -1,16 +1,33 @@
 #!/bin/bash -e
 
-if [ ! -w /app/output ]; then
-    echo "Missing writing permissions on output directory" >&2
-    exit 1
+
+
+if [ "${SEND_REPORT_TO_SERVER}" = true ]; then
+  vars_for_upload=(FHIR_REPORT_DESTINATION_SERVER AUTHOR_IDENTIFIER_SYSTEM AUTHOR_IDENTIFIER_VALUE PROJECT_IDENTIFIER_SYSTEM PROJECT_IDENTIFIER_VALUE)
+  for var in "${vars_for_upload[@]}"; do
+     if [[ -z "${!var}" ]]; then
+        echo "In order to upload the MeasureReport to a FHIR server, all following environment variables must be set" \
+              "(but currently are not set): ${vars_for_upload[*]}"
+        exit 1
+     fi
+  done
+else
+  if [ ! -w /app/output ]; then
+      echo "Missing writing permissions on output directory" >&2
+      exit 1
+  fi
 fi
 
-today=$(date +"%Y-%m-%d_%H-%M-%S")
-measureName="$(jq -c --raw-output '.name' /app/measure.json)"
-outputDir="$today-$measureName"
-mkdir -p /app/output/"$outputDir"
-cp /app/measure.json /app/output/"$outputDir"/measure.json
+now="$(date +%s)"
+dateForDirectory="$(date +"%Y-%m-%d_%H-%M-%S" -d "@${now}")"
+dateForBundle="$(date +"%Y-%m-%dT%H:%M:%S%:z" -d "@${now}")"
```

Comment:
@bastianschaffer  double check if this works - at the moment  when sending it to the fhir server i get:

2024-10-29T09:37:25.991+01:00 ERROR 66065 --- [ main] d.m.f.EvaluationExecutor : java.io.IOException: Failed uploading MeasureReport with status code: '400 BAD_REQUEST' and body: '{"issue":[{"severity":"error","code":"invariant","diagnostics":"Error on value 2024-10-29T09:37:11:z. Expected type is instant, regex ([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T([01][0-9]|2[0-3]):[0-5][0-9]:([0-5][0-9]|60)(\\.[0-9]+)?(Z|(\\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00)).","expression":["entry[0].resource.date"]}],"resourceType":"OperationOutcome"}'